gm4_ink_spitting_squid/data/gm4_ink_spitting_squid/predicates/in_water.json

```diff
@@ -0,0 +1,8 @@
+{
+  "condition": "minecraft:location_check",
+  "predicate": {
+    "fluid": {
+      "fluid": "minecraft:water"
```

Comment:
this should've been `"tag": "minecraft:water"`. `"fluid"` makes it only work with source blocks